<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Engineering</title>
  <link rel="stylesheet" href="../css/public.css">
  <link rel="stylesheet" href="../css/wave.css">
  <link rel="stylesheet" href="../css/engineering.css">
</head>
<body>
  <!-- loading 动画 -->
  <div class="loading-out">
    <div class="loading-fade-in">
        <div class="loading-container">
            <div class="one common"></div>
            <div class="two common"></div>
            <div class="three common"></div>
            <div class="four common"></div>
            <div class="five common"></div>
            <div class="six common"></div>
            <div class="seven common"></div>
            <div class="eight common"></div>
        </div>
    </div>
  </div>
  
  <!-- 头部 -->
  <header>
    <ul>
        <li><a href="../index.html">HOME</a></li>
        <li><a href="#">TEAM</a>
            <dl>
                <dd><a href="../html/member.html">Members</a></dd>
                <dd><a href="../html/attributions.html">Attributions</a></dd>
                <dd><a href="../html/collaborations.html">Collaboration</a></dd>
                <dd><a href="../html/partnership.html">Partnership</a></dd>
            </dl>
        </li>
        <li><a href="#">PROJECT</a>
            <dl>
                <dd><a href="../html/description.html">Description</a></dd>
                <dd><a href="../html/contribution.html">Contributions</a></dd>
                <dd><a href="../html/proof-of-concept.html">Proof of Concept</a></dd>
                <dd><a href="../html/implementation.html">Implementation</a></dd>
                <dd><a href="../html/parts.html">Parts</a></dd>
                <dd><a href="../html/notebook.html">Notebook</a></dd>
            </dl>
        </li>
        <li><a href="#">LAB</a>
            <dl>
                <dd><a href="../html/result.html">Result</a></dd>
                <dd><a href="../html/engineering.html">Engineering</a></dd>
                <dd><a href="../html/model.html">Modeling</a></dd>
            </dl>
        </li>
        <li><a href="../html/safety.html">SAFETY</a></li>
        <li><a href="#">HUMAN PRACTICE</a>
            <dl>
                <dd><a href="../html/overview.html">Overview</a></dd>
                <dd><a href="../html/human-practices.html">Intergrated Human Practice</a></dd>
                <dd><a href="../html/communication.html">Education & Communication</a></dd>
                <dd><a href="../html/sustainable.html">Sustainable</a></dd>
            </dl>
        </li>
        <li><a href="../html/judging.html">JUDGING</a></li>
    </ul>
</header>

  <!-- 滚动条 -->
  <div class="scrollBar">
    <div class="back">BACK</div>
    <div class="bar">
        <img class="lazyload" data-src="../images/main-images/main1.png" alt="">
    </div>
  </div>

  <nav>
    <div class="container">
        <a href="../index.html"><img class="lazyload" data-src="../images/main-images/logo.png" alt=""></a>
        <span class="pic">Engineering</span>
    </div> 
  </nav>
  <div class="wave"></div>

  <div class="steps">
    <h4>Ⅰ Overview</h4>
    <h4>Ⅱ Promoter screening</h4>
    <h4>Ⅲ PTS mutation</h4>
    <h4>Ⅳ Dynamic regulation</h4>
    <h4>Ⅴ Dowmstream-dyeing</h4>
  </div>

  <article>
    <div class="iframe0">
      <h2>Overview</h2>
      <p>Synthetic biology is often described as the design and construction of new biological elements, devices, and systems, as well as the redesign of existing natural biological systems for application purposes (Nature, 2021). As in any other engineering field, synthetic biologists follow iterations of the DBTL cycle (Design, Build, Test, Learn) to successfully develop innovative biological systems (Fig.1).</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/overview-fig1.png" alt="">
        <h5>Fig.1 Iterative design-build-test-learn (DBTL) cycle in Synthetic Biology (Waldby et al. 2018).</h5>
      </div>
      <div class="orange">
        <p><span>Design: </span>Using acquired knowledge either through experimentation or literature, a rational plan is devised for implementing our system.</p>
        <p><span>Build: </span>High efficiency cloning methods are utilized to forge genetic constructs and develop strains to test our systems.</p>
        <p><span>Test: </span>Systems are characterized and quantified with the scientific method.</p>
        <p><span>Learn: </span>Conclusions are drawn on our design and used to inform our next iteration of the cycle.</p>
      </div>
      <p>After an early brainstorming session to identify the problem of "silk spoilage", we wanted to find a bacterial inhibitor that could be applied to silk as a special material. Then we found that many terpenoids were good choices. In the process of researching various terpenoids, we found not only a class of terpenoids with refreshing, antiseptic and antibacterial properties, such as patchoulol and nerolidol, but also many colorful, antioxidant terpenoids, such as lycopene and astaxanthin.</p>
      <p>Because terpenoids are complex compounds, fungi possess more complex organelles for protein expression that promise terpene synthesis. The yeast <em>picha pastoris</em> has become an important industrial microorganism due to its rapid growth, high density fermentation, simple genetic manipulation, and ability to perform post-translational modifications. The sequencing of the whole genome of <em>P.pastoris</em> has led to its application not only as an expression system for exogenous proteins, but also in synthetic biology and metabolic engineering.</p>
      <p>We further understood the biosynthetic pathways of both, and selected <em>P.pastoris</em> as an engineered strain based on the current research status.</p>
      <p>Based on the laboratory conditions and resources, we then chose patchoulol and lycopene, which can be used as bacterial inhibitor and colorant, respectively, in silk staining. Producing them in the same strain reduces the energy consumption and time of fermentation, so we wanted to use a strain with lycopene production pathway already constructed in the lab and introduce the patchoulol synthase gene to achieve co-expression of terpenes. Patchoulol and lycopene share the same precursor FPP(farnesyl pyrophosphate), so, presume FPP stay nearly constant during the same fermentation environment, an increase in the yield of one will result in a corresponding decrease in the yield of the other, further regulating the ratio of the two terpene products by regulating the synthesis of patchoulol as a pathway.</p>
      <p>At the same time, in order to achieve a high yield of both products simultaneously, we performed a screening for the endogenous promoter of <em>P.pastoris</em>; meanwhile, in order to reduce the yield of by-products produced during fermentation by a key enzyme in patchoulol production (patchoulol synthase), we performed a random mutation to achieve an increase in the catalytic activity of the enzyme and its specificity to the substrate. These method above are only static control system. In order to realize dynamic control of both, we introduced a blue light-induced dynamic regulation system to achieve a change in the metabolic flow through a blue light switch to maintain maximum growth of the organism while regulating the product yield.</p>
      <p>In our downstream application, we explored dyeing conditions and create some beautiful handmakes. </p>
      <p>In summary, the introduction of the 4 systems is the result of several engineering design communities. Below are some of the engineering design circles in each system.</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/overview-fig2.png" alt="" style="width:65%;">
      </div>
    </div>
    <div class="iframe1">
      <h2>Cycle Ⅰ-1: the basic strength of promoters</h2>
      <h3>Design: promoter selection</h3>
      <p>There are four commonly used strain types of <em>Pichia pastoris</em> host bacteria (Fig.1), namely X33, GS115, KM71H, and SMD116. Among them, X33 is a wild-type strain, while GS115 and KM71H, both of which have HIS4 nutrient-deficient markers. Strain GS115 has the AOX1 gene and belongs to Mut+, referring to the normal type of methanol utilization; KM71 H strain with AOX1 locus inserted by ARG4 gene belongs to Muts, referring to slow methanol utilization type, and both strains are suitable for general yeast transformation methods. SMD116 (his4 pep4) belongs to the protease deletion type strain, which is able to express some uneven distribution of expression products due to protein degradation. However, the slow growth of SMD116 (his4 pep4) strain resulted in low yield of expressed exogenous protein(Tbl.1). Based on this, we selected strain GS115 as the screening vector for the endogenous promoter.</p>
      <table id="strain">
        <tr>
          <td>Strain</td>
          <td>Genotype</td>
          <td>Phenotype(Pichia only)</td>
        </tr>
        <tr>
          <td>X33</td>
          <td>wild type</td>
          <td>Mut+</td>
        </tr>
        <tr>
          <td>GS115</td>
          <td>his4</td>
          <td>His-,Mut+</td>
        </tr>
        <tr>
          <td>KM71H</td>
          <td>arg4 aox1::ARG4</td>
          <td>Muts,Arg+</td>
        </tr>
        <tr>
          <td>SMD116</td>
          <td>his4 pep4</td>
          <td>Mut+</td>
        </tr>
      </table>
      <h5>Tbl.1 Four strains commonly used by Pichia pastoris host bacteria</h5>
      <p>Through research, we found six constructive promoters and two inducible promoters that are endogenous of <em>P.pastoris</em>. The constructive promoters contain: P<sub><em>ERG9</em></sub>, P<sub><em>GAP</em></sub>, P<sub><em>PGK1</em></sub>, P<sub><em>PET9</em></sub>, P<sub><em>0547</em></sub> and P<sub><em>ACC1</em></sub>. The inducible promoters contain P<sub><em>AOX1</em></sub> and P<sub><em>DAS2</em></sub>. </p>
      <h3>Build: construction of vectors</h3>
      <p>To construct the vectors, the P<sub><em>ERG9</em></sub>, P<sub><em>GAP</em></sub>, P<sub><em>PGK1</em></sub>, P<sub><em>PET9</em></sub>, P<sub><em>0547</em></sub>, P<sub><em>ACC1</em></sub>, P<sub><em>AOX1</em></sub> and P<sub><em>DAS2</em></sub> genes were amplified from genomic DNA of <em>P.pastoris</em>. What's next, we use enhanced green fluorescent protein (EGFP) as report element to characterization the strength of promoters. </p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/1-1.png" alt=""  style="width:45%;">
        <h5>Fig.1 Construction of pPICZA-promoters-EGFP-HIS in GS115.<br>Line 1 are inducible promoters.<br>Line 2 are constructive promoters.</h5>
      </div>
      <h3>Test: the fluorescence intensity</h3>
      <p>The constructed plasmid with promoters and EGFP was subsequently inserted into the yeast genome to build the report system. After incubated at 30°C for 120h, we used microplate reader to test the fluorescence intensity (Fig.2).</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/contribution-images/fluorescence.jpg" alt="">
        <h5>Fig.2 Fluorescence intensity of promoters</h5>
      </div>
      <h3>Learn:</h3>
      <p>After quantifying fluorescence using the microplate reader and measuring od at 600nm, we preliminarily ranked the promoter strength according to the fluorescence per OD (Fig.3). The order of the promoters from strong to weak is: P<sub><em>DAS2</em></sub>, P<sub><em>AOX1</em></sub>, P<sub><em>0547</em></sub>, P<sub><em>GAP</em></sub>, P<sub><em>PET9</em></sub>, P<sub><em>ACC1</em></sub>, P<sub><em>PGK1</em></sub>, P<sub><em>ERG9</em></sub>. The above results also paved the way for us to replace EGFP gene in <em>P.pastoris</em> genome with PTS gene in the next cycle.</p>
      <h2>Cycle Ⅰ-2: productivity of promoters in terpenoids production</h2>
      <h3>Design:</h3>
      <p>Thanks to Xinying Zhang, Ph.D., We got the strain which can already produce lycopene, to be our chassis, and we named it 2OZPP. What's more, 2OZPP here means two copies of gene <em>OcrtE</em> + <em>OcrtB</em> + <em>OcrtI</em> were inserted into the yeast geonomy(Fig.3). </p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/description-images/process.jpg" alt=""  style="width:30%;">
        <h5>Fig.3 Metabolic pathway of FUNCDYES.</h5>
      </div>
      <h3>Build:</h3>
      <div class="rightfloat">
        <div class="float">
          <img class="lazyload" data-src="../images/engineering-images/1-4.png" alt="">
          <h5>Fig.4 Construction of PpICZA-promoters-EGFP-HIS in 2OZPP.</h5>
        </div>
        <p>Based on the strain, we inserted patchoulol synthase (PTS) into the geonomy. Thus, we got eight different stains with eight promoters at the same time(Fig.4).</p>
      </div>
      <h3>Test:</h3>
      <p>After cultured in a shaking incubator under the same conditions for 120 h, we tested the titters of two terpenoids. The quantification of lycopene was performed on a Waters 2695 series HPLC system (Waters Technologies, Milford, MA, USA). The quantification of patchoulol was performed on a GC system (Agilent 7820 a, USA) equipped with HP-5 column (30 m×0.25 mm, 0.25μM U.M film thickness) and a flame ionization detector (FID).</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/proof-of-concept-images/lightbar.jpg" alt="">
        <h5>Fig.5 The production of two terpenoids</h5>
      </div>
      <h3>Learn:</h3>
      <p>We learn from the test above that we constructed the PTS expression pathway successfully. Furthermore, we would try more ways such as PTS mutation to amplify the static control system. And these results would help us to build a rapid selection system of PTS mutations.</p>
    </div>

    <div class="iframe2">
      <h2>CycleⅡ-1: construction of PTS mutation system</h2>
      <h3>Design-1:</h3>
      <p>In vitro selection coupled with directed evolution represents a powerful method for generating nucleic acids and proteins with desired functional properties. Creating high-quality libraries of random sequences is an important step in this process as it allows variants of individual molecules to be generated from a single-parent sequence. These libraries are then screened for individual molecules with interesting, and sometimes very rare, phenotypes. Here, we describe a general method to introduce random nucleotide mutations into a parent sequence that takes advantage of the error-prone polymerase chain reaction (epPCR). Our goal is to realize 2-3bp mutations per 1000bp. What's more, through our research, we found that PTS has poor specificity and produces many by-products during the culture procedure. Thus, we intended to use this method to selected positive mutants which can produce high titer of patchoulol.</p>
      <h3>Build-1:</h3>
      <p>Thanks to our promoter screening, we surprisingly found that there are two promoters that obey a so-called proportional relationship (Fig.1). Once the yeast produces more patchoulol, the production of lycopene reduces. These two promoters are P<sub><em>0547</em></sub> and P<sub><em>PGK1</em></sub>.</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/2-1.png" alt=""  style="width:25%;margin-top:5px;">
        <h5>Fig.1 The relationship between patchoulol and lycopene.</h5>
      </div>
      <p>We chose manganese ions as mutagen for patchouli alcohol synthase gene to produce random mutations through epPCR. In this cycle, we selected 0.4μL, 0.8μL, 1μL and 2μL of 2mM MnSO4 for the first try (Fig.2). The total system for PCR is 50µL.</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/2-2.png" alt="">
        <h5>Fig.2 Construction of PTS mutation vector. </h5>
      </div>
      <h3>Test-1:</h3>
      <p><em>Escherichia coli</em> TOP10F' (Invitrogen, Carlsbad, CA, USA) was used as the host for DNA manipulations and the cells were cultivated in at 37°C. Next, we extracted the DNA from the cell and check the length of fragments through agarose gel electrophoresis.</p>
      <h3>Learn-1:</h3>
      <p>It's so sad that we failed to sequencing. We speculated that the concentration of MnSO<sub>4</sub> is low than the threshold value, so in the next cycle, we would turn up the dosage of MnSO<sub>4</sub>.</p>
      <h3>Design-2 & Learn-2: </h3>
      <p>In this cycle, we selected 1μL, 2μL, 3μL and 4μL of 8mM MnSO4 for the mutation system. At the same time, we use higher concentration for the third try. They are 5μL, 6μL, 7μL and 8μL of 2mM MnSO<sub>4</sub>.</p>
      <p>We failed again. This led us to next cycle of the system by doing loads of research to confirm the mutagen and the concentration.</p>
      <h3>Design-3 & Learn-3:</h3>
      <p>We use 2μL and 6μL of 8mM MnCl<sub>2</sub> instead of 8mM MnSO4 for this cycle.</p>
      <p>To our surprise, we successfully finished the agarose gel electrophoresis and DNA sequencing. However, there are much more mutations than we expected and the template which add 6uL MnCl<sub>2</sub> even fail to combine the primers. So, in the next cycle, we will reduce the usage of mutagen.</p>
      <h3>Design-4 & Learn-4:</h3>
      <p>We use 0.5μL 0.75μL and 1.25μL of 8mM MnCl<sub>2</sub> for this cycle.</p>
      <p>Ultimately, we found that the 0.75uL can realize our goal. Followed by, we would use this concentration to construct our mutation library (Fig.3).</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/2-3.png" alt="">
        <h5>Fig.3 Result of the mutation in cycle 4. The total system for PCR is 50µL. </h5>
      </div>
      <div class="orange">
        <p>Prospect: fixed point mutations</p>
      </div>
      <p>We used modeling method to find the active pocket of PTS(Fig.4). And next step we planned to use fixed point mutations to realize the mutation of these sites. But due to the time limitation, we haven't finished this part of experiment. </p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/2-4.png" alt="">
        <h5>Fig.4 Active pocket of PTS (the yellow part of the picture).</h5>
      </div>
      <h2>Cycle Ⅱ-2: construction of a fast-screening method</h2>
      <h3>Design:</h3>
      <p>Based on the relationship between lycopene and patchoulol production in <em>P.pastoris</em> we have already mentioned, we developed a visual method to screen for efficient mutations in <em>PTS</em>.</p>
      <h3>Build:</h3>
      <p>Firstly, We performed epPCR on <em>PTS</em> at the identified appropriate Mn<sub>2+</sub> concentration and obtained mutated <em>PTS</em> libraries. We have obtained a lycopene producing strain of <em>P.pastoris</em> which names 2OZPP from our advisor Xinying Zhang. Then We completed the construction of mutant <em>PTS</em> yeast library by transferring the mutant <em>PTS</em> into 2OZPP by homologous recombination.</p>
      <h3>Test:</h3>
      <p><em>P.pastoris</em> strain 2OZPP and strains with unmutated <em>PTS</em> constructed in the promoter screening assay were used as controls, and yeast with mutated <em>PTS</em> were inoculated on solid plates and given 1% methanol daily. After 96h of incubation, we observed colonies of different colors (Fig.5).</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/result-images/pts-fig2-min.jpg" alt="" style="width:30%;">
        <h5>Fig.5 First round of PTS visualization screening.</h5>
      </div>
      <h3>Learn:</h3>
      <p>A clear color change of the colonies can be found in the results, and such a screening method is effective. However, we observed that colonies growing on the edges of the plates were generally lighter in color, and we believe that this situation is due to the edge effect caused by nutrient limitation at the edges of the plates, which can mislead the judgment of the <em>PTS</em> mutation effect. Therefore, we optimized it in the next round of DBTL.</p>
      <h2>Cycle Ⅱ-3: edge effect's solution</h2>
      <h3>Design:</h3>
      <p>In order to reduce the misleading effect of edges on the experimental results, we chose a smaller plate in this round of experiments and tried to make the experimental group positioned in the center of the plate.</p>
      <h3>Build & Test:</h3>
      <p>We selected strains that showed lighter colors in the previous round of experiments, re-inoculated them on the plates, and re-arranged the positions of the colonies to obtain better experimental results (Fig.5).</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/2-7.jpg" alt="" style="width:30%">
        <h5>Fig.6 Second round of PTS visualization screening.</h5>
      </div>
      <h3>Learn:</h3>
      <p>We did find some strains that were lighter in color than other transformants, which gave us great confidence. Although we have not quantitatively validated the lighter colored colonies due to time constraints, and we did not find enough positive mutant strains due to limitations in the number of mutant libraries, we are confident that this method must be feasible. </p>
    </div>

    <div class="iframe3">
      <h2>Cycle Ⅲ-1: constructing dynamic regulation with light-control system</h2>
      <p>The idea of constructing a light-control module in our host is derived from our previous wet lab results. As we discovered that by applying mere static control our function of interest cannot be reach, we turn to seek a dynamic control strategy to meet our need of changing the proportion of the two terpenoids at our will.</p>
      <h3>Design:</h3>
      <p>Inspired from certain parts in the database and a recent literature which demonstrated the application of a blue light-induced system in <em>P.pastoris</em>, we establish blue light control system consist of a constitutive or inducible promoter and its downstream gene that encoded light-activated transcriptional factor protein EL222, a promoter including a (C120)<sub>5</sub>  region where EL222 binds to, our target gene and so on. EL222 is expressed as monomer, and turn to dimer as the blue light (465 nm) was on. Then, activated EL222 binds to the (C120)<sub>5</sub> region and initiate the expression of our target gene, <em>PTS</em>. (Fig.1)</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/engineering-images/3-1.png" alt="" style="width:60%">
        <h5>Fig.1 Design of the optogenetic system</h5>
      </div>
      <h3>Build:</h3>
      <p>We synthesize the parts, <a href="http://parts.igem.org/Part:BBa_K4263009" target="_blank">BBa_K4263009</a> and <a href="http://parts.igem.org/Part:BBa_K4263010" target="_blank">BBa_K4263010</a>, and construct them on the same vector pPIcza. Beside, Sequence optimization of our dynamically regulated light-control part <a href="http://parts.igem.org/Part:BBa_K4263009" target="_blank">BBa_K4263009</a> was done to allow it to better play a regulatory role in <em>P.pastoris</em>.(reference to the existed part <a href="http://parts.igem.org/Part:BBa_K4263021" target="_blank">BBa_K4263021</a>)</p>
      <p>Four groups of our host, three parallel tests each, are set with difference in the promoter which control the expression of EL222 in 250ml shake flasks. All of the flasks are cultured for 48h, and their OD were checked every 24h, then methanol was added. </p>
      <p>We also build up a LED device to spray blue light. LEDs (5W, 465 nm) were hung above the flasks to create a blue light environment. The incubator was sealed with black cloth to block the blue light, preventing it from doing harm to the people passing by and reducing the influence from the outside. (Fig.2)</p>
      <div class="twoimg">
        <img class="lazyload" data-src="../images/result-images/dynamic-1-1-min.png" alt=""><img class="lazyload" data-src="../images/result-images/dynamic-1-2-min.png" alt="">
        <h5>Fig.2 The incubator with blue light device</h5>
      </div>
      <h3>Test:</h3>
      <p>The product from each parallel is examined by GC and HPLC. (Fig.3) </p>
      <table id="if3-table">
        <tr>
          <td>Names</td>
          <td>Dark</td>
          <td>Blue light</td>
          <td>Strain</td>
        </tr>
        <tr>
          <td>Control-1</td>
          <td>0</td>
          <td>48h</td>
          <td>2OZPP</td>
        </tr>
        <tr>
          <td>Control-2</td>
          <td>0</td>
          <td>48h</td>
          <td>2OZPP+PTS</td>
        </tr>
        <tr>
          <td>Longer blue light duration</td>
          <td>0</td>
          <td>48h</td>
          <td>2OZPP+PGAP-TF-C(120)5-PTS</td>
        </tr>
        <tr>
          <td>Shorter blue light duration</td>
          <td>16h</td>
          <td>32h</td>
          <td>2OZPP+PGAP-TF-C(120)5-PTS</td>
        </tr>
      </table>
      <div class="twoimg" id="Two">
        <img class="lazyload" data-src="../images/result-images/dynamic-2-2-min.jpg" alt=""><img class="lazyload" data-src="../images/proof-of-concept-images/lightbar.jpg" alt="">
        <h5>Fig.3 Titer and production of lycopene and patchoulol of each parallel</h5>
      </div>
      <h3>Learn:</h3>
      <p>From the results, we are aware that lycopene and patchoulol are produced when blue light is on, which means our blue light-induced system is able to work effectively. Besides, the expression level is related to the dose of the blue light. The longer exposure to blue light, the higher titer of patchoulol. </p>
      <h2>Cycle Ⅲ-2: modifying dynamic regulation with light-control system</h2>
      <h3>Design:</h3>
      <p>Our next question is, naturally, whether the system efficient or not. How can we evaluate the performance and what can we do to improve it? Thus, we use <em>EGFP</em> as the target gene to figure out certain parameters of the system. However, limited time of wet lab disable us from continuing on our design, and we are not ready to present our progress here. The characterization of the parts may be done in the future.</p>
    </div>

    <div class="iframe4">
      <p>After our design, we got the great cell factory that produces FUNCDYES! In the next DBTL cycles, we carried out experiments on dyeing.</p>
      <p>In the industrial production scenario, we hope to use volatile solvent as the extractant to extract patchoulol during fermentation and then concentrate it for dyeing. However, in the laboratory, we used n-dodecane as the extraction agent in the fermentation process, which is not easy to concentrate. Therefore, in the subsequent dyeing experiments, we used store-bought patchoulol for dyeing experiment.</p>
      <p>At the end of fermentation, the fermentation broth was collected and centrifuged to obtain lycopene containing cells, which were then broken to obtain lycopene dissolved in the extractant - ethyl acetate. The dye containing lycopene was obtained by concentrating it for an appropriate time. We added some pure patchoulol to it, and we got our FUNCDYES(Fig.1)! We soaked the silk in FUNCDYES, and after 15 minutes of dyeing, we succeeded in getting the red silk with fragrance!</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/result-images/dowmstream-fig1-min.jpg" alt="" style="width:20%">
        <h5>Fig.1 The fragrant red silk we dyed.</h5>
      </div>
      <p>Next, we experimented with varying the concentration of lycopene to obtain silks of different colors. Also, by changing the ratio of patchouli alcohol to lycopene, we obtained silks with different aromatic intensity and color shades.</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/proof-of-concept-images/igem-min.jpg" alt="" style="width:33%;">
        <h5>Fig.2 We dye different colors of silk.</h5>
      </div>
      <p>We observed the color and fragrance persistence of the silk obtained by dyeing, and we found that the color of the silk lightened slightly over fifteen days, but no significant fading occurred, and the fragrance persisted.</p>
      <p>Although we managed to get fragrant red silks, they are not very bright in color. In the following experiment, we began to think about whether we could achieve better dyeing results by improving the staining conditions.</p>
      <p>We changed the dyeing time, temperature, hoping to find the most suitable dyeing conditions. It was found that prolonging the staining time had no significant effect on the dyeing results, so in the subsequent dyeing experiments, we set the staining time at about 5 minutes. Moreover, the effect of temperature on the dyeing effect is not significant, and considering the cost of temperature control in industry, we believe that dyeing at room temperature is sufficient.</p>
      <p>In addition, we have looked up the common composition of silk dyes on the market and found that most of them are water soluble, while lycopene and patchoulol are hydrophobic substances. Therefore, we set different groups to explore the more appropriate solvent.</p>
      <table id="group">
        <tr>
          <td colspan="3">Grouping of solvent selection experiments</td>
        </tr>
        <tr>
          <td>Group Name</td>
          <td>Solvent</td>
          <td>Additional treatment</td>
        </tr>
        <tr>
          <td>A</td>
          <td>Ethyl acetate</td>
          <td>/</td>
        </tr>
        <tr>
          <td>B</td>
          <td>Ethyl acetate</td>
          <td>Add 0.5 g/mL of NaCl</td>
        </tr>
        <tr>
          <td>C</td>
          <td>Ethyl acetate</td>
          <td>Add 1 g/mL of NaCl</td>
        </tr>
        <tr>
          <td>D</td>
          <td>Ethyl acetate: Water=1:1</td>
          <td>Use ultrasound during dyeing</td>
        </tr>
        <tr>
          <td>E</td>
          <td>Ethyl acetate: Water=1:1</td>
          <td>Add tween 80</td>
        </tr>
        <tr>
          <td>F</td>
          <td>Ethyl acetate: Water=1:1</td>
          <td>Add 0.5 g/mL of NaCl</td>
        </tr>
        <tr>
          <td>G</td>
          <td>Ethyl acetate: Water=1:1</td>
          <td>Add 1 g/mL of NaCl</td>
        </tr>
      </table>
      <h5>Tbl.1 Grouping of solvent selection experiments.</h5>
      <p>It was found that the addition of sodium chloride did have a positive effect on the coloring of the dye, but only in the solvent containing water. However, in solvents containing water, the phenomenon of uneven staining is more likely to occur. The results of the rest of the experimental groups showed that none of them stained as well as the pure ethyl acetate as a solvent. Therefore, we finally chose ethyl acetate as our solvent.</p>
      <div class="oneimg">
        <img class="lazyload" data-src="../images/result-images/dowmstream-fig4-min.jpg" alt="" style="width:23%">
        <h5>Fig.3 Experimental results of staining conditions</h5>
      </div>
      <p>Since we did not get the desired results in our experiments with staining conditions, we came to the staining plant hoping to talk to people with more experience and get some inspiration.</p>
      <p><a href="../html/human-practices.html" target="_blank">Click this link(part one about <strong>Explore the significance</strong>) to see what we communicated~ </a></p>
      <p>After talking with other experienced people, we have suggested some possible experimental improvements in the future. We proposed the idea of using cyclodextrins or liposomes to encapsulate the lycopene and patchoulol molecules, and by modifying the encapsulants, they would chemically react with the amino acids in the silk, thus giving the dyed silk superior color fastness and a longer lasting fragrance. It also occurred to us that it might be possible to use a supercritical CO<sub>2</sub> extractor to dye silk as a solution to the problem that dye molecules are only soluble in organic reagents.</p>
      <p>Although we did not obtain the best experimental results due to technical limitations, we believe that these problems can be optimized in the process of industrial production.</p>
    </div>
  </article>


  <script src="../js/jquery.js"></script>
  <script src="../js/public.js"></script>
  <script src="../js/human-practices.js"></script>
</body>
</html>